docker/local/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: "3.2"
+services:
+ redis:
+    image: "redis:alpine"
+
+    command: redis-server
+    expose:
+     - "6379"
```

Comment:
Is this just exposing it to the docker network?  Maybe we can add a comment here?

Reply:
ugh, I hate docker terminology.
expose == *not* expose, it just tells docker that there is a service running on that port and can only accessed from inside the docker network (so in this case only by the server and the workers, not the host itself or outside)
ports == map docker container ports to the actual hosts.
See https://docs.docker.com/compose/compose-file/compose-file-v3/#expose